package.json

```diff
@@ -82,14 +82,14 @@
     "vitest": "^0.34.6"
   },
   "peerDependencies": {
-    "@prettier/plugin-php": "^0.20.1",
+    "@prettier/plugin-php": ">=0.20.1 <1",
     "@prettier/plugin-ruby": "^4.0.0",
     "@prettier/plugin-xml": "^3.1.0",
     "@xml-tools/parser": "^1.0.11",
     "chevrotain": "7.1.1",
     "prettier": "^3.0.0",
-    "prettier-plugin-glsl": "^0.1.2",
-    "prettier-plugin-sql": "^0.15.0"
+    "prettier-plugin-glsl": ">=0.1.2 <1",
+    "prettier-plugin-sql": ">=0.16.0 <1"
```

Comment:
I think we can still keep `0.15.0` as the minimal version since it's the first version that supports prettier v3: https://github.com/un-ts/prettier/blob/master/packages/sql/CHANGELOG.md#0150

Reply:
done in https://github.com/Sec-ant/prettier-plugin-embed/pull/27/commits/50a16f69cc12cc7a127a1e313d541dcefb6f867f